wikiscrape/export_pages.py

```diff
@@ -0,0 +1,101 @@
+"""Export the pages we enumerated as xml.
+
+This page https://www.mediawiki.org/wiki/Manual:Parameters_to_Special:Export
+lists multiple limits to the amount of data that can be returned. In the two
+wiki's I have been testing on I haven't found these to be true. The main
+points of concern are:
+* pages: The limit is 35
+* limit: The maximum number of revisions to return, limited at 1000.
+* history: It mentions there are cases where this doesn't return all the revisions.
+* listauthors: This didn't seem active on any of the wikis I tested on.
+"""
+
+
+import argparse
+import glob
+import os
+import urllib.parse
+from typing import List
+
+from utils import get_page, get_wiki_name
+
+parser = argparse.ArgumentParser(description="Export mediawikis as XML")
+parser.add_argument("--wiki", required=True, help="The wiki url we are exporting.")
+parser.add_argument(
+    "--pages",
+    action="append",
+    help="A list of files of pages to export, or a dir to export all. defaults to data/${wiki_name}/pages/.",
+)
+# Using firefox I didn't have issues sending a lot of pages at once, but I was
+# getting URI too long errors when using requests.
```

Comment:
Should we specify a user-agent etc.?

Reply:
Do you mean we should pretend to be firefox in case they have some checks for automatic tools?

Reply:
Yes.

Reply:
What about something like this?
```
Mozilla/5.0 (compatible; Licensed-Pile-bot/0.1; +http://www.github.com/r-three/licensed-pile)
```

Would get some goodwill from site owners if we identify as a bot and have a link with info about why we are scraping them?

Reply:
Sounds good 👍